web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -109,6 +109,22 @@ function statusToColor(status: string): string {
   }
 }
 
+function detailedStatusToColor(status: string): string {
```

Comment:
I think coloring should be probably tied to `status` instead of `detailedStatus`, as `detailedStatus` is messaging specific to the type of supervisor, and 'status' is intended to be universal. Values are:
```
    UNHEALTHY_SUPERVISOR
    UNHEALTHY_TASKS
    PENDING
    RUNNING
    SUSPENDED
    STOPPING
```
 See https://github.com/apache/incubator-druid/pull/7428/files#diff-290822ceec7682a31578278f881dad70R58

To me it probably makes sense to use 'status' to set the color, but maybe display the detailed messaging?